README.md

```diff
@@ -222,16 +222,34 @@ npx create-react-app test-template --template file:./carto-react-template/templa
 
 ### Publishing the template
 
-REMEMBER TO COPY the `hygen/_templates` folder.
+> ⚠️ Important: remember to set the right version for each template, tag the release in the GitHub repository and deploy the demo web app with the latest changes
```

Comment:
Maybe the "sample app" instead of the "demo web app" to be consistent with all the docs